src/category_theory/limits/shapes/images.lean

```diff
@@ -273,3 +274,111 @@ end
 end
 
 end category_theory.limits
+
+namespace category_theory.limits
+
+variables {C : Type u} [𝒞 : category.{v} C]
+include 𝒞
+
+section
+
+instance {X Y : C} (f : X ⟶ Y) [has_image f] : has_image (arrow.mk f).hom :=
+by { rw arrow.mk_hom, apply_instance }
```

Comment:
I'm not sure what the term looks like that `rw` creates. It might be safer to use `show` to restate the goal. Alternatively
```suggestion
instance {X Y : C} (f : X ⟶ Y) [h : has_image f] : has_image (arrow.mk f).hom := h
```

Reply:
Good point. I've went with `show`.